### PR TITLE
UIU-2674 Fee/Fine Details to use an ACTIONS menu instead of buttons

### PR DESCRIPTION
**Pull request by iselagb**
Refs. [UIU-2674](https://issues.folio.org/browse/UIU-2674)

UIU-2674

### CODE REVIEW

**Comment by iselagb** [comment]
@mkuklis, Could you review it?, please

File: src/views/AccountDetails/AccountDetails.js [hunk 1]
```diff
@@ -315,6 +319,97 @@ class AccountDetails extends React.Component {
...
   };
 
+  getActionMenu = () => () => {
+    const {
+      account,
+      resources,
+      itemDetails
+    } = this.props;
+
+    const feeFineActions = get(resources, ['feefineactions', 'records'], []);
```
Comment (mkuklis) [inline src/views/AccountDetails/AccountDetails.js:329]: @iselagb we don't need `get` or `_.get`. Please use optional chaining operator: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining

**Review by mkuklis — CHANGES_REQUESTED** [review]

File: src/views/AccountDetails/AccountDetails.js (hunk 1 above)
Comment (iselagb) [reply src/views/AccountDetails/AccountDetails.js]: @mkuklis the changes are already

**Review by mkuklis — APPROVED** [review]
@iselagb thank you for cleaning it up. I will merge this is shortly.

**Comment by mkuklis** [comment]
Closing this one because this work has been merged in: https://github.com/folio-org/ui-users/pull/2257

**Pull request closed without merging**